hazelcast/include/hazelcast/client/protocol/ClientMessage.h

```diff
@@ -642,6 +672,28 @@ class HAZELCAST_API ClientMessage
         return result;
     }
 
+    template<typename T>
```

Comment:
Ditto identical to the function above except the trivial T replaced with `boost::uuids::uuid`.